### PR TITLE
FL: Updated for 2015 session, Rep names fixed

### PR DESCRIPTION
**Pull request by mileswwatkins**
We should be able to turn this scraper back on in full. The representative name issue (Jira bug 369) should be resolved as I'm now gathering Rep names from a more reliable page element.

### CODE REVIEW

**Comment by paultag** [comment]
:+1: thanks!

File: openstates/fl/bills.py [hunk 1]
```diff
@@ -60,13 +60,13 @@ def accept_response(self, response):
                 bill_check and
                 text_check)
         if not valid:
-            raise TimSucksError('Response was invalid. Timsucks.')
+            raise Exception('Response was invalid. Timsucks.')
```
Comment (paultag) [inline openstates/fl/bills.py]: Let's pick a different Exception; ValueError or something (or a new Exception with a better name)

File: openstates/fl/bills.py [hunk 2]
```diff
@@ -60,13 +60,13 @@ def accept_response(self, response):
...
         return valid
 
     def scrape_bill(self, chamber, session, bill_id, title, sponsor, url):
         try:
             html = self.urlopen(url)
-        except TimSucksError:
+        except:
```
Comment (paultag) [inline openstates/fl/bills.py]: This is a nasty heisenbug. Note `except Exception:` and `except:` do different things. Let's change this to match the exception thrown above

**Comment by mileswwatkins** [comment]
@paultag, I'm going to turn this on on Bamboo. The new term hasn't started yet, but there are prefiled bills, and the legislators are up-to-date.

**Comment by paultag** [comment]
OK; go ahead. Turn off the Pupa sync for now, thanks!

On Thu, Feb 5, 2015 at 5:54 PM, Miles Watkins <notifications@github.com>
wrote:

> @paultag <https://github.com/paultag>, I'm going to turn this on on
> Bamboo. The new term hasn't started yet, but there are prefiled bills, and
> the legislators are up-to-date.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunlightlabs/openstates/pull/557#issuecomment-73147708>
> .
>



-- 
Paul Tagliamonte
Software Developer | Sunlight Foundation

**Comment by mileswwatkins** [comment]
But of course! :D

**Pull request merged**